src/packages/@ncigdc/components/Charts/BarChart.js

```diff
@@ -167,6 +168,32 @@ const BarChart = ({
       .on('mouseleave', () => {
         setTooltip();
       });
+  } else {
+    svg
+      .append('text')
+      .attr('class', 'displayOnly')
+      .attr('transform', 'rotate(0)')
+      .attr('y', height + 20)
+      .attr('x', width / 2 - margin.left)
+      .attr('dx', '1em')
+      .style('text-anchor', 'middle')
+      .style('fontSize', '1.2rem')
+      .style('fontWeight', '500')
+      .attr('fill', yAxisStyle.textFill)
+      .text(xAxis.title || '');
+
+    svg
```

Comment:
Should this svg element render at all if there's no `titleForSVG` for it? printing an empty string, and creating the space for it may distort the fit in the svg container (distorting responsiveness).